buildSrc/build.gradle

```diff
@@ -41,25 +55,12 @@ dependencies {
   compile 'de.thetaphi:forbiddenapis:2.0'
 }
 
-Properties props = new Properties()
-props.load(project.file('../gradle.properties').newDataInputStream())
-version = props.getProperty('version')
-
 processResources {
-  inputs.file('../gradle.properties')
-  filter ReplaceTokens, tokens: [
-    'version': props.getProperty('version'),
-    'luceneVersion': props.getProperty('luceneVersion')
-  ]
+  inputs.file('version.properties')
+  from 'version.properties'
 }
 
 extraArchive {
   javadoc = false
   tests = false
 }
-
-eclipse {
-  classpath {
-    defaultOutputDir = new File(file('build'), 'eclipse')
-  }
-}
```

Comment:
and what didnt work, for the record, is the crazy eclipse gradle integration. but it wasn't limited to ignoring this parameter, it basically ignored most of our build logic. I recommend against using it.